examples/webgl_loader_ldraw.html

```diff
@@ -112,68 +113,58 @@
 					separateObjects: false,
 					displayLines: true,
 					conditionalLines: true,
-					smoothNormals: true
+					smoothNormals: true,
+					constructionStep: 0,
+					noConstructionSteps: "No steps."
 				};
 
-				gui = new dat.GUI();
-
-				gui.add( guiData, 'modelFileName', modelFileList ).name( 'Model' ).onFinishChange( function () {
-
-					reloadObject( true );
-
-				} );
+				window.addEventListener( 'resize', onWindowResize, false );
 
-				gui.add( guiData, 'envMapActivated' ).name( 'Env. map' ).onChange( function ( value ) {
+				progressBarDiv = document.createElement( 'div' );
+				progressBarDiv.innerText = "Loading...";
+				progressBarDiv.style.fontSize = "3em";
+				progressBarDiv.style.color = "#888";
+				progressBarDiv.style.display = "block";
+				progressBarDiv.style.position = "absolute";
+				progressBarDiv.style.top = "50%";
+				progressBarDiv.style.width = "100%";
+				progressBarDiv.style.textAlign = "center";
 
-					envMapActivated = value;
 
-					reloadObject( false );
+				// load materials and then the model
 
-				} );
+				reloadObject( true );
 
-				gui.add( guiData, 'separateObjects' ).name( 'Separate Objects' ).onChange( function ( value ) {
+			}
 
-					reloadObject( false );
+			function processConstructionSteps( model ) {
```

Comment:
I like this approach but I wonder if processing the construction steps is better done inside the loader right after the last part is finished loading? And then the total number of steps can go on the root nodes `userData` field?

Reply:
Yes, that seems reasonable. Thanks.

Reply:
I think I'll expose the function `processConstructionSteps` and put the call to it  in `LDrawLoader.load`.

Have you seen my other PR about using `LDrawLoader.parse()` standalone?

Reply:
I've made a commit with your suggestion and renamed `processConstructionSteps` to `computeConstructionSteps`.

Reply:
Looks good to me! Presumably the call to `computeConstructionSteps` would happen in `parse`, as well, once your other PR is merged?

Reply:
You're right! I've made a commit to move the call to `finalizeObject` so it is done in only one place.